lld/test/ELF/bp-section-orderer.s

```diff
@@ -0,0 +1,868 @@
+# REQUIRES: aarch64
+
+# RUN: rm -rf %t && split-file %s %t && cd %t
+# RUN: llvm-mc -filetype=obj -triple=aarch64 a.s -o a.o
+# RUN: llvm-profdata merge a.proftext -o a.profdata
+
+.ifdef GEN
```

Comment:
I think you cannot use the simplified `.ifdef GEN` style if you are running more commands in this test.